core/src/stored/backends/unix_file_device.d/bareos-dir.d/storage/Filechanger.conf.example.in

```diff
@@ -0,0 +1,10 @@
+Storage {
+  Name = File # Not named "Filechanger" so that it can replace the default single File storage
```

Comment:
If you change again the Name, then it destroy the whole coherency of the examples
Then why not renaming also the File to File.conf.example so it can really replace the default ? But then you also have to change the whole just written documentation to again make it coherent, explicit and actionable)

To me this commit will again just make people at the end confused again.

If at the end, they really want to call it `File` while it is a virtual file changer, that's no more our concern.